app/views/user_mailer/delete_work_notification.text.erb

```diff
@@ -1,9 +1,12 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
-  
-Your work "<%= @work.title.html_safe %>" was deleted at <%= (@work.pseuds.count > 1 && @user != User.current_user) ? "the request of " + text_pseud(User.current_user.default_pseud) : "your request" %>.
-  
-If you have questions, please contact Support at <%= root_url %>support.
-  
-Attached is a copy of your work for your reference.
-<% end %>
\ No newline at end of file
+<%= t('.hello', user:  @user.default_pseud.byline) %>
+<%= (@work.pseuds.count > 1 && @user != User.current_user) ?
+t('.part1_other',
+        title: @work.title,
+        pseud: User.current_user.default_pseud) :
+t('.part1_yourself',
+        title: @work.title) %>
+<%= t('.part2', support: t('.support')+' '+new_feedback_report_url ) %>
```

Comment:
There's a space before the `)` that shouldn't be there.

And I just want to make sure I understand -- this will take the string for the `support` key, add a space after it, and then write the Support URL? So for the plain text email, `If you have questions, please %{support}.` will be `If you have questions, please contact Support http://ao3.org/support.` while the HTML email will be  `If you have questions, please <a href="http://ao3.org/support">contact Support</a>.`?